src/bento/armDetailData.js

```diff
@@ -50,7 +50,7 @@ const subsections = [
 
 // Ideal size for externalLinkIcon is 16x16 px
 export const externalLinkIcon = {
-  src: 'https://raw.githubusercontent.com/CBIIT/bento-frontend/master/src/assets/program/externalLinkIcon.svg',
+  src: 'https://raw.githubusercontent.com/CBIIT/datacommons-assets/main/bento/images/icons/svgs/externalLinkIcon.svg',
```

Comment:
Can you also remove these existing files from this branch and also can you clean up unused assets in this repo.